native/acter/api.rsh

```diff
@@ -2337,6 +2357,12 @@ object Activity {
     /// content of this activity (e.g. comment), if any
     fn msg_content() -> Option<MsgContent>;
 
+    /// the details of this membership change activity
+    fn membership_content() -> Option<MembershipContent>;
+
+    /// the details of this profile change activity
+    fn profile_content() -> Option<ProfileContent>;
+
```

Comment:
why not keep the `membership_change` and expose that information through that? Would there ever be a `profile_content` without a `membership_content`?

Reply:
The current structure is suitable to generate the room state change text.
If profile change and membership change are integrated, it would be hard to generate state text.